src/js/components/organisms/colors/stories/colors.md

```diff
@@ -0,0 +1,4 @@
+# Colors
+
+## DSM
```

Comment:
could link to these instead with the current caveat:
below are links to the dsm guidelines, however, the current implementation does not include the tooltip show in the guidelines
* [small](https://ultimaker.invisionapp.com/dsm/ultimaker/ultimaker-com/asset/components/5c4af1b9dd18e10fa57787f7)
* [medium](https://ultimaker.invisionapp.com/dsm/ultimaker/ultimaker-com/asset/components/5c4b0f186b659fc43d6d11b3)
* [large](https://ultimaker.invisionapp.com/dsm/ultimaker/ultimaker-com/asset/components/5c4b13a4198d49e1eea97263)